src/Rest/TikaRestClient.php

```diff
@@ -39,12 +50,16 @@ public function isAvailable()
             $result = $this->get(null);
             $result->setAuth($this->options['username'], $this->options['password']);
             $result->send();
+
             if ($result->getResponse()->getStatusCode() == 200) {
                 return true;
             }
         } catch (RequestException $ex) {
-            SS_Log::log(sprintf("Tika unavailable - %s", $ex->getMessage()), SS_Log::ERR);
-            return false;
+            var_dump(sprintf("Tika unavailable - %s", $ex->getMessage()), SS_Log::ERR);
+            die;
```

Comment:
```php
use Psr\Log\LoggerInterface;
// ...
catch (RequestException $ex) {
    Injector::inst()->get(LoggerInterface::class)->error(sprintf("Tika unavailable - %s", $ex->getMessage());
    return false;
}
```

Reply:
If you could remove this debug statement then we're in a reasonable position to merge this as a first pass

Reply:
How embarrassing. Did I mention I was busy yesterday? :-P

Done!